frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/option_action.dart

```diff
@@ -18,6 +18,24 @@ enum OptionAction {
   moveDown,
   color,
   divider,
+  align,
+}
+
+enum OptionAlignType {
+  left,
+  center,
+  right;
+
+  String get name {
+    switch (this) {
+      case OptionAlignType.left:
+        return 'left';
+      case OptionAlignType.center:
+        return 'center';
+      case OptionAlignType.right:
+        return 'right';
+    }
+  }
```

Comment:
```dart
enum OptionAlignType {
  left('left'),
  center('center'),
  right('right');

  const OptionAlignType(this.name);
  
  final String name;
}
```

using an enhanced enum might make this more readable.

Reply:
IIRC, if you just need the name and nothing else, you should be fine using this:
https://stackoverflow.com/a/63491617